client/ayon_resolve/plugins/create/create_shot_clip.py

```diff
@@ -1,23 +1,143 @@
-# from pprint import pformat
+import copy
+
 from ayon_resolve.api import plugin, lib, constants
 from ayon_resolve.api.lib import (
     get_video_track_names,
+    get_current_timeline_items,
     create_bin,
 )
 from ayon_core.pipeline.create import CreatorError, CreatedInstance
 from ayon_core.lib import BoolDef, EnumDef, TextDef, UILabelDef, NumberDef
 
 
+# Used as a key by the creators in order to 
+# retrieve the instances data into clip markers.
+_CONTENT_ID = "resolve_sub_products"
+
+
+class _ResolveInstanceCreator(plugin.HiddenResolvePublishCreator):
```

Comment:
@iLLiCiTiT should this "base class" be taken out of this file so the Plug-in discovery system does not even find it and starts thinking it's a creator it needs to "process" in one way or another being just bloat to the plug-in registry at that point?

Reply:
It can be. But I have nothing agains keeping it here if this is the only file using it.